.vscode/settings.json

```diff
@@ -7,7 +7,7 @@
     "**/lib": true,
     "**/dist": true
   },
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
Why revert this? I've found it to be super useful.

Reply:
Accidentally got it in my commit. I always have to turn it off when editting templates and XML files because it always totally messes up the files,